modules/index-service/src/main/java/org/opencastproject/index/service/impl/index/event/Event.java

```diff
@@ -98,6 +90,7 @@
   private static final Map<String, String> recordingStatusMapping = new HashMap<>();
 
   static {
+    recordingStatusMapping.put(RecordingState.SCHEDULED, "EVENTS.EVENTS.STATUS.SCHEDULED");
```

Comment:
@lkiesow Newly introduced status to distinguish an event supposed to be a scheduled event (managed by scheduler) vs. archived event. Replaces status dependency on scheduling status.